client/js/verification-portal/community-health-center/chcInputResource.js

```diff
@@ -0,0 +1,45 @@
+export const chcInputResource = [
```

Comment:
@bhavyaab The file name should be all lowercase, as it can cause issues in remote environments (dev/stage/prod)